src/dlgTriggerEditor.cpp

```diff
@@ -196,13 +195,42 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
 
     mudlet::loadEdbeeTheme(mpHost->mEditorTheme, mpHost->mEditorThemeFile);
 
+    edbee::StringTextAutoCompleteProvider* provider = new edbee::StringTextAutoCompleteProvider();
+    //QScopedPointer<edbee::StringTextAutoCompleteProvider> provider(new edbee::StringTextAutoCompleteProvider);
+
+    // Add lua functions and reserved lua terms to an AutoComplete provider
+    for(QString key : mudlet::mLuaFunctionNames.keys())
+    {
+        provider->add(key, 3, mudlet::mLuaFunctionNames.value(key).toString());
+    }
+
+    provider->add("and", 14);
+    provider->add("break", 14);
+    provider->add("else", 14);
+    provider->add("elseif", 14);
+    provider->add("end", 14);
+    provider->add("false", 14);
+    provider->add("for", 14);
+    provider->add("function", 14);
+    provider->add("local", 14);
+    provider->add("nil", 14);
+    provider->add("not", 14);
+    provider->add("repeat", 14);
+    provider->add("return", 14);
+    provider->add("then", 14);
+    provider->add("true", 14);
+    provider->add("until", 14);
+    provider->add("while", 14);
```

Comment:
Let's add `goto` here given that we know that's coming in 5.3

Reply:
Can doooo

Reply:
Um, I thought we were stuck on Lua 5.1.x because changing that would break backwards compatibility - or are we planned a step-change/break with Mudlet **5.0** (or, um, **4.0**)?

Reply:
No plans, but can't hurt to hint to avoid that as a variable name